workflow-gaps.md

```diff
@@ -0,0 +1,66 @@
+- Workflow Gaps
+    - Communciation with/from api/cms team
+      - As of now the communcation from the above teams leaves a lot to be desired. this is specially true regarding breaking changes.
+      - Api/cms team should communicate breaking changes/etc to us and vice versa.
+      - Action Items:
+        - ?
+
+    - Documentation
+      - As of now the folder structure in the project has no documentation
+      - As of now none of the logic on the mobile client is documented
+        - This will be essential as we move forward with OneBuilder and other abstractions that we've added. Specially since we have new devs joining.
+      - Action Items
+        - Document new abstractions that we've added
+          - OneBuilder
+          - Smart reducers
+          - Sagas
+          - etc
+        - Document our consulting company friend's legacy code (would be nice to do if we had infinite time)
+
+    - Single source of truth for standards
+      - As of now there is no single source of truth for standards
+      - Standards should be documentated
+      - There exists a standards document for the web client, however mobile client would benefit from keeping its own standards document
+      - Action Items:
+        - Decide on plan for documentating standards
+        - Create standards document (or not based on action item above)
+
+    - Code Review
+      - As of now code review process is not being followed (obviously due to deadlines)
+      - Need to ensure that code review comments are productive
+      - [Great resource for ensuring that code review is productive ](https://mtlynch.io/human-code-reviews-1/#settle-style-arguments-with-a-style-guide)
+      - [Part 2](https://mtlynch.io/human-code-reviews-2/#offer-sincere-praise)
+        - TLDR:
+          - Be proactive with examples
+          - Criticize the work not the person
+          - Tie comments to principles
+            - "We should split this class in two"
+            - vs
+            - "We should split this class in two, because performance blah blah blah"
+          - Phrase feedback as request not commands
+            - Can we/Shouldnt this/Wouldn't a better way/etc
+          - Prefer commenting on architecture over style
+            - abstract style to a styleguide/linter
+
+    - Linting
```

Comment:
We 10000% need a linter. It should follow AirBnb style guide. We don't have a 'pipeline' per sey, because this is mobile, but should be able to be added to each dev's code editor and should be enforced at the PR level.

Reply:
We should still be able to add a lint command on each PR even if we don't have a pipeline right?